source/cloud-native-platform/guides/afd-waf.html.md.erb

```diff
@@ -22,7 +22,7 @@ You can find out what the blocked rule is by following these steps:
 
 ```
 AzureDiagnostics 
-| where ResourceProvider == "MICROSOFT.NETWORK" and Category == "FrontdoorWebApplicationFirewallLog"
+| where ResourceProvider == "MICROSOFT.CDN" and Category == "FrontDoorAccessLog"
```

Comment:
this isn't right you've put the access log, this is fore the WAF

Reply:
ah - updated now